src/features/contributor-activity/components/ErrorState.tsx

```
@@ -0,0 +1,17 @@
+interface ErrorStateProps {
+  message: string;
+  onRetry?: () => void;
+}
+
+export const ErrorState: React.FC<ErrorStateProps> = ({ message, onRetry }) => {
+  return (
+    <div className="error-state">
+      <p className="error-message">{message}</p>
+      {onRetry && (
+        <button onClick={onRetry} className="btn-retry">
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)